test/dynamo/test_generator.py

```diff
@@ -1053,6 +1223,149 @@ def fn(t):
         fn(t)
 
 
+class GeneratorThrowCpythonTests(GeneratorTestsBase):
+    # Taken from commit
+    # https://github.com/python/cpython/blob/d51a4ca1123e3e49e5cae4273355bdfd9e419a10
+    # changed the tests a little bit to run them inside dynamo
+    # + replaced all self.assert* calls to plain assert statements
+
+    @unittest.expectedFailure
```

Comment:
are the expectedFailures intentional?

Reply:
Yes. They need `CLEANUP_THROW`, which is implemented in the next PR.